### DISCUSSION TITLE
Print Dialog Stalling on Chrome if no footer icons are specified.

### DISCUSSION THREAD

**Discussion by kenkaijie** [pradyunsg/furo#561] [Potential Issue] [answered]
Thank you for this theme!

I have encountered an issue when trying to print some documentation via Chrome.

![image](https://user-images.githubusercontent.com/14245153/196585997-1dd1d10c-3f34-4b64-891a-3f9c9d05de95.png)

The issue occurs when `conf.py` is not set up with `footer_icons` in `html_theme_options`. If no icon is configured, it seems the presence of the `<div class="icons></div>` empty nested flex box causes the print dialog to have a very large page count.

This is related (in symptom) to #358.

**Comment by pradyunsg** [comment, marked as answer]
Thanks for a clear description on this @kenkaijie! ^>^

308cf677 should resolve this.
